packages/react/src/executors/module-federation-dev-server/module-federation-dev-server.impl.ts

```diff
@@ -38,9 +41,24 @@ export default async function* moduleFederationDevServer(
     );
   }
 
+  const patternMatchedRemotesToSkip = new Set(
+    findMatchingProjects(options.skipRemotes, context.projectGraph.nodes)
+  );
+  const remotesNotInGraph = options.skipRemotes.filter((r) => {
+    const { exclude, type } = parseStringPattern(r, context.projectGraph.nodes);
+    return exclude === false && type === 'unlabeled';
+  });
```

Comment:
Is there a way to do this check without using `parseStringPattern`?

Can we just check if it's a node in the graph?

Reply:
I was thinking more about this, hence the note in the description. We could try only run the pattern matching on the remotes that are in the config, then check if any of those that match are in the graph. 

But i still need a function to do that pattern matching, and it seems like `parseStringPattern` is the function that does that? 

I could duplicate the logic?